ckanext/versioning/logic/action.py

```diff
@@ -455,3 +456,9 @@ def _get_dataset_name(id_or_name):
         raise toolkit.ObjectNotFound('Package {} not found'.format(id_or_name))
 
     return dataset.name
+
+
+def is_valid_revision_id(revision_ref):
+    if is_hex_str(revision_ref, chars=32):
+        return True
+    return False
```

Comment:
Why not just:
```suggestion
    return is_hex_str(revision_ref, chars=32)
```
`¯\_(ツ)_/¯`

Reply:
(note that this change is not needed if we just remove this code and rely on new metastore-lib functionality)